### PR TITLE
Check for duplicate rows

### PR DESCRIPTION
**Pull request by JedGrabman**
### Description
New validation check raises warning if files contain multiple copies of an identical row (which indicates possible data integrity issues)

### Changelog
- validate.py - Added `check_duplicate_rows` and related calls
- test_checks.py - Added tests for `check_duplicate_rows` in validate.py
- PLANS.md - Remove duplicate rows task from list

### Fixes 
- Fixes "check for duplicate rows" task in PLANS.md (no associated Github issue)

### CODE REVIEW

**Comment by JedGrabman** [comment]
@nmdefries for review.

**Review by nmdefries — CHANGES_REQUESTED** [review]
This works when I run the validator, but tests don't pass due to missing params.

File: validator/PLANS.md [hunk 1]
```diff
@@ -38,7 +38,6 @@
 
 ### Starter/small issues
 
-* Check for duplicate rows
```
[the remark this answers is not in the document]
Comment (JedGrabman) [reply validator/PLANS.md:41]: Done

File: validator/tests/test_checks.py [hunk 2]
```diff
@@ -228,6 +228,41 @@ def test_invalid_geo_id_national(self):
         assert "US" not in validator.raised_errors[0].expression
         assert "SP" not in validator.raised_errors[0].expression
 
+class TestDuplicatedRows:
+    params = {}
```
[the remark this answers is not in the document]
Comment (JedGrabman) [reply validator/tests/test_checks.py]: Copied `params` from another test, see if that resolves it. If it's possible to set reasonable defaults for any of these values, we should add that to our TODOs.

**Review by nmdefries — APPROVED** [review]

File: validator/tests/test_checks.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (nmdefries) [reply validator/tests/test_checks.py]: That's a thought. Initially I intentionally didn't define defaults for a small set of params that greatly change the outcome of running this package (what dataset being used, dates, etc).

**Comment by krivard** [comment]
Should this go into `main` or the branch?

**Comment by nmdefries** [comment]
@krivard This should go into main; I changed the base branch to reflect that.

**Pull request merged**